modules/tinymce/src/core/test/ts/browser/bookmark/BookmarksTest.ts

```diff
@@ -223,4 +223,50 @@ describe('browser.tinymce.core.bookmark.BookmarksTest', () => {
     assertApproxRawContent(editor, '<p>abc</p>');
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
   }));
+
+  context('Get bookmark should work if selection is on a comment', () => {
+    const hook = TinyHooks.bddSetupLight<Editor>({
+      base_url: '/project/tinymce/js/tinymce'
+    }, [], false);
+
+    const outsideButton: SugarElement<HTMLButtonElement> = SugarElement.fromHtml('<button id="getBookmarkButton">Get Bookmark</button>');
+    const setSelectionToCommentAndTriggerGetBookmark = (editor: Editor, content: string) => {
+      editor.resetContent(content);
+      editor.addCommand('getBookmarkProxyCommand', () => {
+        editor.selection.getBookmark();
+      });
+
+      const clickHandler = (): void => {
+        editor.execCommand('getBookmarkProxyCommand');
+      };
+      outsideButton.dom.addEventListener('click', clickHandler);
+
+      outsideButton.dom.click();
+      outsideButton.dom.removeEventListener('click', clickHandler);
+    };
+
+    before(() => {
+      Insert.append(SugarBody.body(), outsideButton);
+    });
+
+    after(() => {
+      Remove.remove(outsideButton);
+    });
```

Comment:
These should be before the above functions (after the editor hook) if you decided to keep this test.

Reply:
Alternatively... you might be able to use the `TinyHooks.bddSetupFromElement` function instead for this test because you could setup the elements and keep it all connected.

Reply:
Ok, I refactored with `TinyHooks.bddSetupFromElement`, I think it would be good to keep these tests since they are more "e2e" than the one in `NormalizeRangeTest`